### PR TITLE
Elizabeth edits for Chap 1

### PR DESCRIPTION
**Pull request by E-Ream**
**Tasks for** @E-Ream:

Welcome to Jupyter Notebooks and Vocareum
- [x] Proof read
- [x] Keep autograder tests in this notebook, add a sentence or two explaining that Vocareum is running code like this behind the scenes to check the solution

Learning Python Basics
- [x] Proof read
- [ ] Remove autograder tests
- [ ] Hide solutions
- [x] Change title to "Python Basics I: Variables, Strings, and Bugs"
- [x] Draft learning objectives (few bullet points)

Lists, Dictionaries, Enumeration
- [x] Change name to "Python Basics III: Lists, Dictionaries, and Enumeration"
- [x] Proof read
- [ ] Remove autograder tests
- [ ] Hide solutions
- [x] Add example for enumerating over Dictionary
- [x] Draft learning objectives (few bullet points)

Linear Algebra with Numpy and Scipy
- [x] Proof read
- [ ] Remove autograder tests
- [ ] Hide solutions

Manipulating Data with Pandas
- [x] Remove reference from Sakai. Instead, just link to the publisher's website.
- [x] Proof read
- [ ] Remove autograder tests
- [ ] Hide solutions

Functions as Arguments
- [x] Restructure the notebook:
   - [x] Create a simpler example showing a function taking a function as an input
   - [x] Create simpler examples (and activities) showing lambda functions
   - [x] Make the midpoint integration a long example at the end of the notebook
- [x] Proof read
- [x] Remove 4c-ii from one of the headings
- [ ] Remove autograder tests
- [ ] Hide solutions

Functions, Scoping, and Other Fun Stuff
- [x] Restructure and split into 3 notebooks:
   - [x] Functions and Scope (4a)
   - [x] Recursion (4b)
   - [x] Files and Modules (4c)
- [x] Add example notebook by merging high/low guessing game at end of notebook with tutorial 1
- [x] Proof read
- [ ] Remove autograder tests
- [ ] Hide solutions

### CODE REVIEW

**Comment by E-Ream** [comment]
@adowling2 Quick question, for the home activities that involve copying code from above and changing it or writing new code, do you want this to be done already before it's published?  The Chap 1-02 Variables notebook has some of them already completed but I wasn't sure if this was on purpose.

**Comment by E-Ream** [comment]
@adowling2 @jkeane14 How should we go about updating the table of contents in each branch?  I edited and committed the changes to the toc with in the chapter 2 branch but it does not show in the other branches.  Should we just edit the table of contents in one branch for the entire time?

**Comment by adowling2** [comment]
Does this make sense?

**Comment by E-Ream** [comment]
> Does this make sense?

Yes it does, I'll be sure to do that.

**Comment by adowling2** [comment]
> 



> > @adowling2 When there are instructions referring to gradescope or working with a partner, do you want us to take that out or leave it in? I attached an example. image
> 
> @adowling2 @jkeane14 Along the same line, if there are any references to bringing pseudocode to class or any other references to a classroom setting, should we remove it?

This is a great question. I would rephrase the reference to having the pseudocode next to you. Can you post the specific text here? I'll then show how I would rephrase it.

**Comment by jkeane14** [comment]
@E-Ream  Suggestions/Edits for Chap 1:

Notebook 4:
- [ ] In the motivating example after the first block of code one line starts with mathematical, was this supposed to be mathematically? 
- [ ] After the lines of equations there is a line that says addition twice in a row
- [ ] Under "Return Values" section in the block of code, the comments say x and y are both said to be the first value returned

Notebook 4c:
- [ ] Text for sphere appears weird - some lines highlighted but I can't tell why 

Notebook 5:
- [ ] Learning objective says vaule instead of value
- [ ] Bananas is spelled wrong in shopping list example

Notebook 6:
- [ ] In the paragraph above "Creating arrays in neat ways", the main takeaway says "why in doubt" but should this be "when in doubt"?
-In the home activity "Write a sentence with at least 50 letters..." is this sentence supposed to come prefilled? Also in this line behavior is spelled wrong

Notebook 8:
- [ ] I could not import matplotlib.pyplot without error, this is probably a me problem but thought I would mention it just in case
-In Portfolio Calculator section first line, computer is spelled wrong

Notebook 9:
- [ ] I had the matplotlib.pyplot error again so I could not test graphs. Everything else looked good but I couldn't get to the solution you added for the class activity under convergence analysis...
